models/reviews.models.js

```diff
@@ -5,3 +5,21 @@ exports.selectCategories = () => {
     return result.rows;
   });
 };
+
+exports.selectReviews = () => {
+  return db
+    .query(
+      `
+      SELECT owner, reviews.review_id, title, category, designer, review_img_url, reviews.votes, reviews.created_at, COUNT(*) AS comment_count FROM reviews
+      LEFT JOIN comments on reviews.review_id = comments.review_id
+      GROUP BY reviews.review_id
+      ORDER BY created_at DESC;
+  `
+    )
+    .then((result) => {
+      result.rows.forEach((review) => {
+        review.comment_count = Number(review.comment_count);
```

Comment:
Nice type coercion, take a look at the CAST .. AS Function - if we're thinking ahead and there'd be a LOT of comments, this way of changing the type of each comment count could take a looong time.
CAST .. AS allows as us to change the type directly in the SQL Query 👀